syserol/model.py

```diff
@@ -51,7 +51,7 @@ def cross_validation():
         matrix[test_index, 0:6] = copy[test_index, len(glycan) : len(glycan) + 6]  # store original value
         copy[test_index, len(glycan) : len(glycan) + 6] = np.nan  # artificially make the value NaN
 
-        _, matrixFac, _ = perform_CMTF(cube, copy, 2)  # run decomposition on new matrix
+        _, matrixFac, _ = perform_CMTF(cube, copy, 10)  # run decomposition on new matrix
```

Comment:
Make the number of components an argument to the function. You may want to run this for varied numbers.